ndcube/utils/cube.py

```diff
@@ -164,14 +164,16 @@ def get_crop_item_from_points(points, wcs, crop_by_values, keepdims):
                 point_indices_with_inputs.append(i)
                 array_axes_with_input.append(point_inputs_array_axes[i])
         array_axes_with_input = set(chain.from_iterable(array_axes_with_input))
-        array_axes_without_input = set(range(wcs.pixel_n_dim)) - array_axes_with_input
+        array_axes_without_input = set(range(low_level_wcs.pixel_n_dim)) - array_axes_with_input
         # Slice out the axes that do not correspond to a coord
         # from the WCS and the input point.
-        wcs_slice = np.array([slice(None)] * wcs.pixel_n_dim)
-        if len(array_axes_without_input):
+        if len(array_axes_without_input) > 0:
+            wcs_slice = np.array([slice(None)] * low_level_wcs.pixel_n_dim)
             wcs_slice[np.array(list(array_axes_without_input))] = 0
-        sliced_wcs = SlicedLowLevelWCS(wcs, slices=tuple(wcs_slice))
-        sliced_point = np.array(point, dtype=object)[np.array(point_indices_with_inputs)]
+            sliced_wcs = SlicedLowLevelWCS(low_level_wcs, slices=tuple(wcs_slice))
+            sliced_point = np.array(point, dtype=object)[np.array(point_indices_with_inputs)]
+        else:
+            sliced_wcs, sliced_point = low_level_wcs, np.array(point, dtype=object)
```

Comment:
I suggest adding a comment here that the variable `sliced_wcs` won't actually be a sliced WCS in this logic branch.  Just saving the potential confusion for a future maintainer.